sdk/openai/Azure.AI.OpenAI/src/Custom/Assistants/AzureRunStepDetailsUpdate.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Diagnostics.CodeAnalysis;
+
+namespace Azure.AI.OpenAI;
+
+public static class RunStepDetailsUpdateExtensions
```

Comment:
ah, I missed this -- we need to remove (or at least [EditorBrowsable(Never)]) these Assistants files since the matching feature was briefly postponed.

Reply:
I'll remove all these types.